### DISCUSSION TITLE
Getting far different colors on rescale

### DISCUSSION THREAD

**Discussion by wes-kay** [ImageMagick/ImageMagick#5224] [Help]
I'm using `mogrify -unsharp 0.25x0.25+8+0.065 -resize 256x256 -normalize -density 180 *.png` to rescale all my images down from 400px to 256px. The issue is I'm getting major differences in shading as well as colors.

![Screenshot_14](https://user-images.githubusercontent.com/40011470/173687920-5c9b0619-4925-4085-b8d2-6c1062a72fc7.png)
resized (left image) is darker, is there any other commands to try and get it to the most accurate image? As the shadows are unbalanced.

**Comment by fmw42** [comment]
I do not see any color changes. Your use of -normalize will change the effective contrast more in the dark areas than the light areas since it is effectively -contrast-stretch 2%x1%. If you want even change then use -level with equal amounts on dark and light ends (e.g. -level 5x95% or perhaps -contrast-stretch 1%x1%. But even -contrast-stretch by itself could be uneven, depending upon the histogram. You could also try -auto-level -contrast-stretch 1%x1% or whatever values you like. Note, generally, one would do the unsharp after the resize. But if you are happy with your result, that is OK also.

**Comment by snibgo** [comment]
Looking carefully at the pixel values, in areas with little detail, the shadows on the left are slightly darker (by a couple of percent) than the shadows on the right. And the highlights on the left are slightly lighter than the highlights on the right.

This is exactly what we should expect from "-normalize". If you don't like it, then don't "-normalize".
